lib/timescaledb/rails/models/chunk.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Timescaledb
+  module Rails
+    # :nodoc:
+    class Chunk < ::ActiveRecord::Base
+      self.table_name = 'timescaledb_information.chunks'
+      self.primary_key = 'hypertable_name'
+
+      scope :compressed, -> { where(is_compressed: true) }
+      scope :decompressed, -> { where(is_compressed: false) }
+
+      def full_name
```

Comment:
Maybe to follow how they name things, let's call it `chunk_full_name`, what do you think?